hack/generate-manifest-flow-aggregator.sh

```diff
@@ -20,9 +20,11 @@ function echoerr {
     >&2 echo "$@"
 }
 
-_usage="Usage: $0 [--mode (dev|release)] [--keep] [--help|-h]
+_usage="Usage: $0 [--mode (dev|release)] [-fc|--flow-collector] [--keep] [--help|-h]
 Generate a YAML manifest for flow aggregator, using Kustomize, and print it to stdout.
         --mode (dev|release)  Choose the configuration variant that you need (default is 'dev')
+        --flow-collector      Flow collector is the externalFlowCollectorAddr configMap parameter
+                              It show given in format IP:port:proto. Example: 192.168.1.100:4739:udp
```

Comment:
"It show given"?

do you mean "it should be given"?

Reply:
not fixed yet

Reply:
Fixed in one script but missed it in the other one. Done now.